module/data/abstract.mjs

```diff
@@ -341,9 +341,10 @@ export class ActorDataModel extends SystemDataModel {
    * @param {object} [options]
    * @param {boolean} [options.deterministic] Whether to force deterministic values for data properties that could be
    *                                          either a die term or a flat term.
+   * @param {boolean} [options.extended]      Include extra values like "name" that aren't useful in formulas.
```

Comment:
It felt a little off to me to have an option that adds data to `getRollData` that is not useful for rolls, so I was going to suggest maybe `getReferenceData` that calls `getRollData` and then adds supplementary data on top. But actually, there's probably a lot of stuff under `system` that is not useful for *rolls* specifically, and we include it all.

I don't really know what else we would include as `extended` data. There are only fields that exist at the top level, and there aren't many of those (some we already include, like `flags` now), as well as some derived properties like `statuses` which we may end up including anyway (#3122).

So maybe we should just get rid of the `extended` option and always include `name`?